ui/StatusQ/src/formatteddoubleproperty.cpp

```diff
@@ -0,0 +1,66 @@
+#include "StatusQ/formatteddoubleproperty.h"
+
+#include <QDebug>
+
+FormattedDoubleProperty::FormattedDoubleProperty(QObject* parent)
+    : QObject{parent}
+{
+    m_locale.setNumberOptions(QLocale::DefaultNumberOptions | QLocale::OmitGroupSeparator);
+}
+
+QVariant FormattedDoubleProperty::value() const
+{
+    return m_value;
+}
+
+void FormattedDoubleProperty::setValue(const QVariant& newValue)
+{
+    if (!newValue.isValid()) {
+        qWarning() << "Setting property to invalid value:" << newValue << "is not supported";
```

Comment:
Suggestion: Maybe you could provide more explicit waring info by providing the `.typeName()` value.

Reply:
Fixed